lib/desired-caps.js

```diff
@@ -176,6 +176,9 @@ let desiredCapConstraints = _.defaults({
   wdaEventloopIdleDelay: {
     isNumber: true
   },
+  otherApps: {
```

Comment:
Add this in https://github.com/appium/appium-xcuitest-driver#desired-capabilities :)